pkg/utils/k8sutil.go

```diff
@@ -528,6 +537,8 @@ func getInitImage(runtime string) string {
 		return "bitnami/ruby:2.4"
 	case strings.Contains(runtime, "dotnetcore"):
 		return "microsoft/aspnetcore-build:2.0"
+	case strings.Contains(runtime, "bash"):
```

Comment:
I don't think we should use an init image for the bash runtime. Note that system packages are installed in different places so we cannot mount a single path into the runtime container. For this case I think we should install dependencies at runtime. As explained in the Dockerfile using a entry point script.

Reply:
Also, note that in the line 1002 we should change the condition `if funcObj.Spec.Deps != "" {` with `if funcObj.Spec.Deps != "" && funcObj.Spec.Deps != "bash"  {`